CMakeLists.txt

```diff
@@ -90,7 +112,8 @@ install(
   PATTERN  "*.hpp"
   PATTERN "test*" EXCLUDE
   PATTERN ".git" EXCLUDE
+  PATTERN "${_excluded_headers}" EXCLUDE
```

Comment:
I am not sure if empty string is properly treated as no-op. Can I assume you verified this?

Reply:
Correct, they are ignored. This will be a noop.